internal/services/netapp/validate/volume_group_sap_hana_volumes_export_policy_validation.go

```diff
@@ -0,0 +1,30 @@
+package validate
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/netapp/2022-05-01/volumegroups"
+)
+
+func ValidateNetAppVolumeGroupExportPolicyRuleSAPHanna(rule volumegroups.ExportPolicyRule, protocolType string) []error {
```

Comment:
We should add a TODO to revisit this when multiple protocols are supported

Reply:
For this one, we will need to skip since this is not even planned yet, whenever (and if) this comes up, this will be updated for sure. I don't want to add a TODO so customers looking at this code may think that is coming at anu point in time and Microsoft might have some issues managing expectations.